entrypoint

```diff
@@ -24,6 +24,11 @@ if [ -n "$INPUT_CTESTFLAGS" ]; then
     ctest+=( "$INPUT_CTESTFLAGS" )
 fi
 
+cmake=( cmake )
+if [ -n "$INPUT_CMAKEFLAGS" ]; then
+    cmake+=( $INPUT_CMAKEFLAGS )
+fi
```

Comment:
I did some digging, can you try the following?

    IFS=' ' read -r -a cmakeflags <<< "$INPUT_CMAKEFLAGS"
    cmake+=( "${cmakeflags[@]" )

This creates a proper `cmakeflags` arrays, and then concatenates to the `cmake` array

Reply:
I just implemented that in the latest push, it works as expected.